### PR TITLE
added detection for IPv4

### LINKED ISSUE #7: Implement IPv4 Address Detection
**Description:** A user's IP address is one of the most critically important pieces of information a user has to protect while online. Thief of the IP address can lead to location and information leaks that will be used by attackers to gain access to crucial information that the address holder owns

Example:
XXX.XXX.XXX.XXX or 123.321.456.654
_X in this case represents a digit for each of the 4 sections_

Detection Steps: the users can use numerical symbols to put in place detection strategies that will ensure correct handling and usage of their IP address. The Python code will identify a user's IP by examining each, up to 3 digit, section of their IP addresses. If the code reads IP examples that have 4 digits sections or has none numeric symbols it will not be considered a valid IP address. 
**Valid example:** 123.321.456.654
 **Nonvalid example:** @31.5673.9T2.1O1

### PR DESCRIPTION
**Pull request by aalston2**

### CODE REVIEW

**Review by daseanneclos — APPROVED** [review]

File: test_pii_scan.py [hunk 1]
```diff
@@ -38,6 +38,23 @@ def test_political_group_detection(self):
...
         self.assertNotIn('NRP', str(results))
 
+    def test_IPv4_address_detection(self):
+        # test for a valid IPv4 address. (no alphabet or special characters)
+        results = analyze_text('123.123.123.123')
+        print(results)
+        self.assertIn('IP_ADDRESS', str(results))
+
+        results = analyze_text('!&^*.R#S.123.TY$')
+        self.assertNotIn('NRP', str(results))
```
Comment (claesmk) [inline test_pii_scan.py]: I think you meant to say IP_ADDRESS here not NRP

**Review by claesmk — CHANGES_REQUESTED** [review]
I still see NRP which should be IP_ADDRESS

File: test_pii_scan.py (hunk 1 above)
Comment (aalston2) [reply test_pii_scan.py]: thank you for letting me know I will adjust this mistake

**Review by claesmk — APPROVED** [review]
Looks good

**Review by nsmiller1 — APPROVED** [review]
Looks fine to me. I'm curious though why only one self.assertIn for the results

**Comment by claesmk** [comment]
@aalston2 this looks ready for you to merge and delete your branch and Codespace

**Pull request merged**